data/sacd.attestation.example.json

```diff
@@ -0,0 +1,37 @@
+{
+    "specversion": "1.0",
+    "timestamp": "2025-03-11T14:30:00Z",
+    "type": "dimo.sacd",
+    "data": {
+        "grantor": {
+            "address": "0x1110000000000000000000000000000000000000",
+            "name": "Optional Human Readable Name"
+        },
+        "grantee": {
+            "address": "0x2220000000000000000000000000000000000000",
+            "name": "Optional Human Readable Name"
+        },
+        "effectiveAt": "2025-03-11T14:30:00Z",
+        "expiresAt": "2030-12-20T05:20:45Z",
+        "asset": "did:nft:137:0x3330000000000000000000000000000000000000_31415",
```

Comment:
Shouldn't it be inside each agreement?

Reply:
This is the structure we were discussing in the PRD https://docs.google.com/document/d/1LSge1IWZGj6moUw4esajpu217SHG304jYpxeDKxk8hE/edit?tab=t.9lkww3mbrv47

I believe the reasoning is that any attestation agreement is only going to apply to a single vehicle nft

Reply:
I do agree about the effecitveAt/ expiresAt, as I've thought about it more since that initial meeting. I'd be in favor of eliminating it from either the agreement section or this parent section for attestations, since i think it mgith make it duplicative